runtime/cudaq/platform/default/rest/helpers/oqc/OQCServerHelp.cpp

```diff
@@ -280,6 +301,17 @@ OQCServerHelper::processResults(ServerMessage &postJobResponse,
     }
   }
 
+  if (outputNames.find("output_names." + jobId) == outputNames.end()) {
+    throw std::runtime_error("Could not find output names for job " + jobId +
+                             " this " + std::to_string((long)this));
```

Comment:
This will print a memory address in base-10? What would a user get from that, I wonder.

Reply:
Probably not much. I think that is a holdover from old debug. For anybody who reads this, the server helper is destroyed after submitting the job and reconstructed later when grabbing the results. (That was the motivation for the debug because I couldn't figure out why it wasn't retaining values that I had saved to the object!) I'll remove that because I have since resolved the issue that I was debugging at the time.

Reply:
Updated in c4fc0b8a